File: src/tool/spring-boot-generator/output/authority/authority/src/main/java/com/authority/pojo/entity/RoleSystemUserInfo.java
package com.authority.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;



@ApiModel("role system user")
@Getter
@Setter
public class RoleSystemUserInfo  {

    @ApiModelProperty("account")
    private String account;

    @ApiModelProperty("password")
    private String password;

    @ApiModelProperty("create time")
    private Date createTime;

    @ApiModelProperty("")
    private String role;

}

File: src/tool/spring-boot-generator/output/authority/db/roleSystemUserInfo.js
let roleSystemUserInfo = {
	name: "roleSystemUserInfo",
	description: "role system user",
	columns: {
		account: {
			name: "account",
			description: "account",
			isPk: true,
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		password: {
			name: "password",
			description: "password",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "create time",
			nullable: false,
			type: {
				name: "timestamp",
				length: null
			}
		},
		role: {
			name: "role",
			description: "",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"account",
	"password",
	"createTime",
	"role"
];

function create() {
  return {};
}

roleSystemUserInfo.columnsArray=columnsArray;
roleSystemUserInfo.create=create;
module.exports=roleSystemUserInfo;